modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -973,70 +973,86 @@ definition {
 
 		var slaName = "SLAName";
 
-		WorkflowMetrics.goToWorkflowSLASettings(workflowProcessName = "Single Approver");
+		task ("Given a blog entry with an SLA and the account language is set to English") {
+			WorkflowMetrics.executeIndexActions(actionDescription = "Workflow Indexes");
 
-		Click(locator1 = "WorkflowMetricsSLA#ADD_NEW_SLA_BUTTON");
+			WorkflowMetrics.goToWorkflowSLASettings(workflowProcessName = "Single Approver");
 
-		WorkflowMetrics.fillSLAForm(
-			slaDefinitionDays = 1,
-			slaDefinitionDescription = "SLA Description",
-			slaDefinitionEnd = "Process Ends: Approved",
-			slaDefinitionHours = 0000,
-			slaDefinitionName = ${slaName},
-			slaDefinitionStart = "Process Begins");
+			Click(locator1 = "WorkflowMetricsSLA#ADD_NEW_SLA_BUTTON");
 
-		Button.clickSave();
+			WorkflowMetrics.fillSLAForm(
+				slaDefinitionDays = 1,
+				slaDefinitionDescription = "SLA Description",
+				slaDefinitionEnd = "Process Ends: Approved",
+				slaDefinitionHours = 0000,
+				slaDefinitionName = ${slaName},
+				slaDefinitionStart = "Process Begins");
 
-		Workflow.openWorkflowProcessBuilderConfiguration();
+			Button.clickSave();
 
-		Workflow.configureWorkflow(
-			workflowDefinition = "Single Approver",
-			workflowResourceValue = "Blogs Entry");
+			Workflow.openWorkflowProcessBuilderConfiguration();
 
-		JSONBlog.addEntry(
-			entryContent = "Test Content",
-			entryTitle = "Test Blog");
+			Workflow.configureWorkflow(
+				workflowDefinition = "Single Approver",
+				workflowResourceValue = "Blogs Entry");
 
-		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+			JSONBlog.addEntry(
+				entryContent = "Test Content",
+				entryTitle = "Test Blog");
 
-		Click(locator1 = "WorkflowMetrics#DASHBOARD_TOTAL_PENDING_ITEMS_TITLE");
+			WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
 
-		WaitForElementPresent(locator1 = "WorkflowAllItems#WORKFLOW_INSTANCE_ID");
+			Click(locator1 = "WorkflowMetrics#DASHBOARD_TOTAL_PENDING_ITEMS_TITLE");
 
-		var getInstanceId = selenium.getText("WorkflowAllItems#WORKFLOW_INSTANCE_ID");
+			Pause(locator1 = 80000);
 
-		Click(locator1 = "WorkflowAllItems#WORKFLOW_INSTANCE_ID");
+			while ((IsElementNotPresent(locator1 = "WorkflowAllItems#DETAIL_PAGE_SLA_NAME")) && (maxIterations = "25")) {
+				Refresh();
 
-		WaitForPageLoad();
+				Click(locator1 = "WorkflowAllItems#WORKFLOW_INSTANCE_ID");
+			}
```

Comment:
The pause and iterations were added so that the SLA has time to process. There isn't a solid set number that I've figured out yet, but I do plan on eventually going back and refactoring this part.